### PR TITLE
updated to add remaining addresses

### PR DESCRIPTION
**Pull request by cryptorex**
added remaining BTC, LTC, ETH, ZEC addresses for both categories

### CODE REVIEW

File: tracker.js [hunk 1]
```diff
@@ -1,24 +1,86 @@
...
-    var BtczBalance1_toBtc = (BtczBalance1*BtczValue);
-    var BtczBalance2_toBtc = (BtczBalance2*BtczValue);
+
+
     document.getElementById('statsTxt').innerHTML = "<b>" + totalDonations + " BTCZ</b> raised in <b>" + hours + " hours</b>, average: <b>" + hourlyAverage + " BTCZ/h</b>";
-    document.getElementById('btczBalance1').innerHTML = BtczBalance1.toFixed(0) + " BTCZ";
+
+	
+	var BtczBalance1_toBtc = (BtczBalance1*BtczValue);
+	document.getElementById('btczBalance1').innerHTML = BtczBalance1.toFixed(0) + " BTCZ";
```
Comment (egyptianbman) [inline tracker.js:32]: Good ol' `document.getElementById`.

Comment (cryptorex) [reply tracker.js:32]: old school - just following the scheme, respecting the previous coder ;)

Comment (egyptianbman) [reply tracker.js:32]: haha, yeah. Good stuff. Taking me back to the good ol' days.

Comment (adrianoferraguto) [reply tracker.js:32]: I like it more than using jQuery or other stuff

**Pull request merged**